services/core-web/src/components/noticeOfWork/applications/FinalPermitDocuments.js

```diff
@@ -152,39 +150,33 @@ export class FinalPermitDocuments extends Component {
             <PermitPackage isAdminView={this.props.adminView} />
           </div>
         </div>
-        {isNoWApplication && (
+        {(this.props.showPreambleFileMetadata && this.props.showInUnifiedView && (
+          <FormSection name="final_requested_documents_metadata">
+            {unifiedDocumentsView}
+          </FormSection>
+        )) ||
+          unifiedDocumentsView}
+        {!this.props.showInUnifiedView && (
           <>
-            {(this.props.showPreambleFileMetadata && this.props.showInUnifiedView && (
-              <FormSection name="final_requested_documents_metadata">
-                {unifiedDocumentsView}
+            <h4>Original Documents</h4>
+            <p>These documents came in with the original application.</p>
+            {(this.props.showPreambleFileMetadata && (
+              <FormSection name="final_original_documents_metadata">
+                {nowSubmissionDocuments}
               </FormSection>
             )) ||
-              unifiedDocumentsView}
-            {!this.props.showInUnifiedView && (
-              <>
-                <h4>Original Documents</h4>
-                <p>These documents came in with the original application.</p>
-                {(this.props.showPreambleFileMetadata && (
-                  <FormSection name="final_original_documents_metadata">
-                    {nowSubmissionDocuments}
-                  </FormSection>
-                )) ||
-                  nowSubmissionDocuments}
-                <br />
-
-                <h4>Requested Documents</h4>
-                <p>
-                  These documents were added after the original application but were provided by the
-                  proponent.
-                </p>
-                {(this.props.showPreambleFileMetadata && (
-                  <FormSection name="final_requested_documents_metadata">
-                    {nowDocuments}
-                  </FormSection>
-                )) ||
-                  nowDocuments}
-              </>
-            )}
+              nowSubmissionDocuments}
+            <br />
+
+            <h4>Requested Documents</h4>
+            <p>
+              These documents were added after the original application but were provided by the
+              proponent.
+            </p>
+            {(this.props.showPreambleFileMetadata && (
+              <FormSection name="final_requested_documents_metadata">{nowDocuments}</FormSection>
+            )) ||
+              nowDocuments}
           </>
         )}
       </div>
```

Comment:
It looks like a lot of this was deleted, but it was formatting. Here is the actual block of code without the "IsNoWApplication" check

```
<div>
        <div className="inline-flex between">
          <div>
            {!this.props.adminView && <h4>Permit Package</h4>}
            <p>All files in this list will appear in the Preamble on the permit.</p>
          </div>
          <div>
            <PermitPackage isAdminView={this.props.adminView} />
          </div>
        </div>
        {(this.props.showPreambleFileMetadata && this.props.showInUnifiedView && (
          <FormSection name="final_requested_documents_metadata">
            {unifiedDocumentsView}
          </FormSection>
        )) ||
          unifiedDocumentsView}
        {!this.props.showInUnifiedView && (
          <>
            <h4>Original Documents</h4>
            <p>These documents came in with the original application.</p>
            {(this.props.showPreambleFileMetadata && (
              <FormSection name="final_original_documents_metadata">
                {nowSubmissionDocuments}
              </FormSection>
            )) ||
              nowSubmissionDocuments}
            <br />

            <h4>Requested Documents</h4>
            <p>
              These documents were added after the original application but were provided by the
              proponent.
            </p>
            {(this.props.showPreambleFileMetadata && (
              <FormSection name="final_requested_documents_metadata">{nowDocuments}</FormSection>
            )) ||
              nowDocuments}
          </>
        )}
      </div>
      ```